app/exchanges/volume_tracking/message_builder.rb

```diff
@@ -80,8 +80,7 @@ def populate_by_used_type(data)
     end
 
     def used_by_well_barcode
-      "#{object.used_by.plate.sequencing_kit_box_barcode}:" \
-        "#{object.used_by.plate.plate_number}:#{object.used_by.position}"
+      "#{object.used_by.run.name}:#{object.used_by.position}"
```

Comment:
Apologies. It looks like I got this slightly wrong in the story. We will still need the plate number otherwise we would not be able to identify them uniquely.

Reply:
Yeah makes sense, will add it back 👍